.github/workflows/labels.yml

```diff
@@ -105,6 +105,7 @@ jobs:
 
                   log('Last updated at', pull_request.updated_at)
                   if (new Date(pull_request.updated_at) < cutoff) return done()
+                  log('URL', pull_request.html_url)
```

Comment:
I nearly commented _"should these logs be combined to avoid race conditions"_, then I remembered this is concurrent code not parallel code; so there's no race conditions unless we yield (e.g. with another `await`)

Some of the other logs later on (after yielding with `await`) could run into race conditions though... But, Looking closer at your `log` function, race conditions are handled by prefixing all logs with `pull_request.number`. So this entire comment was basically pointless :see_no_evil:

Reply:
Unrelated: what's the thought process behind _always_ printing the "updated at" date, but only printing the URL for _matching_ PRs?